how-to/business-logic/how-to-change-user-role.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How to Change User Role Programatically
+description: An example on changing the user role programatically, using cloud code
+type: how-to
+page_title: How to Change User Role Programatically
+slug: bl-change-user-role
+position:
+tags: backendservices, business-logic, cloud-code, user-role, cloud-functions
+teampulseid: 
+ticketid: 1016776
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+You need to change/set a user's role programatically based on a certain logic (without going to {{site.tp}} portal).

```

Comment:
"Frequently you need to set programmatically the role of a user account programmatically based on certain logic. .....For example,  when the user achieves certain rank or performs an in-app purchase...... This requires administrative privileges for the app and can be achieved with server-side logic." .....